python312Packages.playwrightcapture: 1.26.3 -> 1.27.0, python312Packages.lacuscore: 1.11.3 -> 1.12.0

fabaff opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 353963`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>python311Packages.lacuscore</li>
    <li>python311Packages.lacuscore.dist</li>
    <li>python311Packages.playwrightcapture</li>
    <li>python311Packages.playwrightcapture.dist</li>
    <li>python312Packages.lacuscore</li>
    <li>python312Packages.lacuscore.dist</li>
    <li>python312Packages.playwrightcapture</li>
    <li>python312Packages.playwrightcapture.dist</li>
  </ul>
</details>

NickCao commented on pkgs/development/python-modules/playwrightcapture/default.nix:
```diff
@@ -22,7 +22,7 @@
+  version = "1.27.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```
```suggestion
  disabled = pythonOlder "3.9";
```

NickCao commented on pkgs/development/python-modules/lacuscore/default.nix:
```diff
@@ -17,7 +17,7 @@
+  version = "1.12.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```
```suggestion
  disabled = pythonOlder "3.9";
```

NickCao commented:
@ofborg eval

fabaff committed:
python312Packages.playwrightcapture: 1.26.3 -> 1.27.0

Diff: Lookyloo/PlaywrightCapture@refs/tags/v1.26.3...v1.27.0

Changelog: https://github.com/Lookyloo/PlaywrightCapture/releases/tag/v1.27.0

fabaff committed:
python312Packages.lacuscore: 1.11.3 -> 1.12.1

Diff: ail-project/LacusCore@refs/tags/v1.11.3...v1.12.1

Changelog: https://github.com/ail-project/LacusCore/releases/tag/v1.12.1

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 353963`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>python311Packages.lacuscore</li>
    <li>python311Packages.lacuscore.dist</li>
    <li>python311Packages.playwrightcapture</li>
    <li>python311Packages.playwrightcapture.dist</li>
    <li>python312Packages.lacuscore</li>
    <li>python312Packages.lacuscore.dist</li>
    <li>python312Packages.playwrightcapture</li>
    <li>python312Packages.playwrightcapture.dist</li>
  </ul>
</details>
